e2e_tests/utils.py

```diff
@@ -98,3 +109,16 @@ def a_project_name(project_slug: str):
     "Environment",
     ["workspace", "project", "user_token", "admin_token", "admin", "user"],
 )
+
+
+def reinitialize_container(sys_id: str, container_type: str, project: str):
```

Comment:
I'd say both `initialize_container`and `reinitialize_container` should be in the same module (`utils.py` sounds nice)

Reply:
Done